vhive.go

```diff
@@ -81,9 +83,17 @@ func main() {
 	isLazyMode = flag.Bool("lazy", false, "Enable lazy serving mode when UPFs are enabled")
 	criSock = flag.String("criSock", "/etc/firecracker-containerd/fccd-cri.sock", "Socket address for CRI service")
 	hostIface = flag.String("hostIface", "", "Host net-interface for the VMs to bind to for internet access")
+	gvisor := flag.Bool("gvisor", false, "Use gvisor sandboxes instead of firecracker MicroVMs")
```

Comment:
replace with `-sandbox` that can be: firecracker or gvisor and nothing else. add an assertion. firecracker is default (stock doesn't need to be here as the vhive daemon is not running for that)